Makefile

```diff
@@ -186,6 +199,29 @@ test-gitlab2:
 	source configs/rtl.sh; \
 	cd ../tests && plptest --threads 16 --stdout
 
+# Bender integration
+
+VLOG_ARGS += -suppress 2583 -suppress 13314
+BENDER_BUILD_DIR = sim
+
+.PHONY: bender-script bender-build bender-rm
+bender-script: 
```

Comment:
I think it would make sense to have the `bender` target as a dependency of the script target. Like this, bender would be automatically downloaded upon first use of the bender-script. What do you think?

Reply:
This makes sense. I didn't add it, as the `checkout` command would call it, with `bender-script` serving as a support target, and for the logical flow I decided to separate dependency download and script generation, but bender will fetch the dependencies if they're not there, so ensuring bender is present for this command is reasonable, will add.